src/utils/sources-tree/getDirectories.js

```diff
@@ -16,7 +16,7 @@ function findSource(sourceTree: TreeDirectory, sourceUrl: string): TreeNode {
         _traverse(child);
       }
     } else if (!returnTarget) {
-      if (subtree.path.replace(/http(s)?:\//, "") == sourceUrl) {
+      if (subtree.path.replace(/http(s)?:\/\//, "") == sourceUrl) {
```

Comment:
we should add a test for getDirectories that reproduces the bug 
https://github.com/jbhoosreddy/debugger.html/blob/sourcemap-self-url-unique/src/utils/sources-tree/tests/utils.spec.js#L155

also, i wonder why this line was written this way, is it handling a case we're missing?
could we re-write it to be more general with getURL?